zorg/buildbot/builders/SphinxDocsBuilder.py

```diff
@@ -176,6 +176,12 @@ def getSphinxRuntimesDocsBuildFactory(
             src_to_build_dir='runtimes',
             **kwargs) # Pass through all the extra arguments.
 
+    f.addStep(ShellCommand(
+                name="Install pip depenencies",
+                command=['pip install -r llvm-project/llvm/docs/requirements.txt'],
```

Comment:
This might have to be relative...I probably want `src_dir` instead of `obj_dir`.

Reply:
8184e7fd11a5